dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -457,6 +458,17 @@ const mobileStickyAdStyles = css`
 	}
 `;
 
+const mobileStickyAdStylesFullWidth = css`
+	width: 100%;
+	text-align: center;
+	background-color: ${palette.neutral[97]};
+
+	.ad-slot[data-label-show='true']::before {
+		padding-left: calc((100% - 320px) / 2);
+		padding-right: calc((100% - 320px) / 2);
```

Comment:
Will this still look okay for [300 x 50px sticky ads](https://github.com/guardian/commercial/pull/1403)?

Reply:
I haven't really covered the different sizes and I just applied the fixed width as it's currently been used in PROD but good spot on Simon! 

I guess that change in 7752a6f87737c9fdf85e0515df07227385389662 fixes it. It is hard to test the `300x50` because the adtest doesn't have a creative with that size but it works as expected for `320px` so I would assume it will work for `300px` as well. 

I might ask Alex on Mon to add that creative size to the test just to be sure it works as expected.